modules/administration_manual/pages/installation/deployment_considerations.adoc

```diff
@@ -124,7 +124,7 @@ supported with ownCloud Enterprise Edition only.
 While many customers are starting with NFS, sooner or later that
 requires scale-out storage. Currently the options are GPFS or GlusterFS,
 or an object store protocol like S3 (supported in Enterprise Edition
-only) or Swift. S3 also allows access to Ceph Storage.
+only). S3 also allows access to Ceph Storage.
```

Comment:
The text: `(supported in Enterprise Edition only)` needs to be removed because S3 is now for all editions.
To check see: https://marketplace.owncloud.com/search/s3

Reply:
I think it's correct that way, in the first line we specified "customers", the community tar does not include those apps.

Reply:
@cdamken this is correct that the apps are not included in the tarball, but you can download it from marketplace without any restriction to EE only. Therefore EE only is wrong. It was like that in the past, but has changed. This needs to be reflected by removing that wrong statement.

Reply:
<img width="1086" alt="grafik" src="https://user-images.githubusercontent.com/25486818/51302813-c27ae900-1a33-11e9-9ee2-bdffdd086a11.png">

But the "old" S3 App ist still Enterprise - so it´s still kind of true :)

Reply:
So, what's the resolution on this one?

Reply:
The old S3 is depriciated in favor the new one, is in the process to be phased out and not longer maintained...